src/Controller/Api/FileController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Controller\Api;
+
+use App\Entity\File;
+use App\Message\DeleteFile;
+use App\Repository\FileRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * CRUD API for File Entity.
+ */
+class FileController extends EntityController
```

Comment:
I don't think we should be extending EntityController anymore. It's based around entityHandler, we should take care of our own security and permissions. You can extend the AbstractFOSRestController if so desired.

Reply:
Changed that to use AbstractFOSRestController